pkg/build/graph/analysis/bc.go

```diff
@@ -17,6 +18,7 @@ import (
 const (
 	MissingRequiredRegistryWarning = "MissingRequiredRegistry"
 	MissingImageStreamWarning      = "MissingImageStream"
```

Comment:
split this into two constants.  One for build input and another for build output.